### ISSUE TITLE
[BUG] - Glitched parts of E3D models

### ISSUE THREAD

**Issue by JezSonic** [MaSzyna-Reloaded/MaSzyna-API-wrapper#108]
### Description

Some meshes like shown on screenshot below are not sticked properly ot the rest of loco
![Godot_v4 3-stable_win64_5B1tF5laA1](https://github.com/user-attachments/assets/5612014b-7310-4db5-80da-f2dfe4de0655)

Also as you may notice they also have not smooth but triangled curves that cause shadows to look weird on them
![Godot_v4 3-stable_win64_nqhzkzghcb](https://github.com/user-attachments/assets/1a8c5b0e-cdac-4f72-9d8e-3e869bf7422b)

### Reproduction steps

```bash
1. Go to demo on `main`
2. Run it
3. See glitches
```

### OS

Windows 64bit

**Comment by marcinn** [comment]
Yes, it looks bad. Godot renderer may give different results due to different lighting and shaders/materials, but we should investigate. Note that the original exe file isn't perfect either.

![image](https://github.com/user-attachments/assets/846314b0-1195-40d3-a391-a1cf5f8ce2f5)

**Comment by marcinn** [comment]
There are more things to check. 

1. I feel that normal maps aren't handled properly.
2. There could be still something wrong with normals or tangents.

No GI (looks quite good, but notice the triangle under the nose)
![image](https://github.com/user-attachments/assets/5f5810bb-2378-46f3-8195-5cfb6e1732b3)

With GI (shadow looks incorrect on the face, and the triangle under the nose is too bright):
![image](https://github.com/user-attachments/assets/a6006322-ea48-413c-90ba-b8e4d7361b2a)
